cmd/rocsp-tool/main.go

```diff
@@ -141,10 +142,15 @@ func main2() error {
 			}
 			parsed, err := ocsp.ParseResponse(resp, nil)
 			if err != nil {
-				logger.Infof("parsing error on %x: %s", resp, err)
+				fmt.Fprintf(os.Stderr, "parsing error on %x: %s", resp, err)
```

Comment:
switching from the logger to os.Stderr here leaves only two uses of the logger, on lines 175 and 178. Is it even worth keeping around at that point?

Alternatively, maybe errors like these should still go through the logger.

Reply:
I think it's still worthwhile for the long-running tasks (load-from-db and scan-responses). And InitWrappedDb wants a logger argument that I'd rather not map to /dev/null.